ckanext/switzerland/dcat/profiles.py

```diff
@@ -170,44 +208,63 @@ def parse_dataset(self, dataset_dict, dataset_ref):  # noqa
         # Resources
         for distribution in self._distributions(dataset_ref):
 
-            resource_dict = {'media_type': None}
+            resource_dict = {
+                'media_type': None,
+                'language': [],
+            }
 
             #  Simple values
             for key, predicate in (
+                    ('identifier', DCT.identifier),
                     ('format', DCT['format']),
                     ('mimetype', DCAT.mediaType),
                     ('media_type', DCAT.mediaType),
                     ('download_url', DCAT.downloadURL),
                     ('access_url', DCAT.accessURL),
-                    ('issued', DCT.issued),
-                    ('modified', DCT.modified),
                     ('rights', DCT.rights),
-                    ('license', DCT.license),
+                    ('license', DCT.rights),
```

Comment:
This is wrong, license != rights, these are two separate fields. For opendata.swiss only rights is used, but it's possible to harvest sources with license and rights attributes.